mod/ckeditor/views/default/js/ckeditor/boot.js

```diff
@@ -0,0 +1,16 @@
+define(function (require) {
+	var $ = require('jquery');
+
+	return {
+		addBehavior: function (context) {
+			$longtexts = $('.elgg-input-longtext:not([data-cke-init])', context);
+			if ($longtexts.length) {
+				require(['elgg/ckeditor'], function(elggCKEditor) {
```

Comment:
Kinda wish there were a promise-based API for this so you can put all requires at top but not block on them:

```js
var $ = require('jquery'); // block module executing on jquery
var elggCKEditorPromise = require(['elgg/ckeditor']); // don't block, but start loading now.

// ...
elggCKEditorPromise.then(function(elggCKEditor) { // at the point we need it, wait for promise to resolve first
  $longtexts.ckeditor(elggCKEditor.init, elggCKEditor.config);
});
```